app/controllers/analysis_page_finder_controller.rb

```diff
@@ -12,9 +11,15 @@ def show
 
     alert_type = find_advice_page_of_class(analysis_class)
     if alert_type && alert_type.advice_page.present?
-      redirect_to advice_page_path(school, alert_type.advice_page)
+      redirect_to advice_page_path(school, alert_type.advice_page), status: :moved_permanently
```

Comment:
Added status code to force crawlers to update indexes. These links were only present in the old version of the comparison pags.